simplq/src/serviceWorker.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable */
```

Comment:
I've disabled linting in this file. Any idea what this file is about?

Reply:
It's for enabling local caching on the final site. It's not enabled right now because it'll make development harder with changes not showing up on saves. We'll enable after the whole new UI is done. You don't need to disable linting though. Could you look at new-frontend? I'm pretty sure I fixed all the errors by adding global rules instead of inline rules when it makes sense.

Reply:
let me check